doc/conf.py

```diff
@@ -26,13 +26,14 @@
 
 autodoc_mock_imports = ['deepspeech']
 
-read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
-if read_the_docs_build:
-    import subprocess
-    subprocess.call('cd ../ && npm install typedoc@0.17.4 typescript@3.8.3 @types/node@13.9.x', shell=True)
-    subprocess.call('cd ../ && doxygen doc/doxygen-c.conf', shell=True)
-    subprocess.call('cd ../ && doxygen doc/doxygen-java.conf', shell=True)
-    subprocess.call('cd ../ && doxygen doc/doxygen-dotnet.conf', shell=True)
+# This is in fact only relevant on ReadTheDocs, but we want to run the same way
+# on our CI as in RTD to avoid regressions on RTD that we would not catch on
+# TaskCluster
+import subprocess
+subprocess.call('cd ../ && npm install typedoc@0.17.4 typescript@3.8.3 @types/node@13.9.x', shell=True)
+subprocess.call('cd ../ && doxygen doc/doxygen-c.conf', shell=True)
+subprocess.call('cd ../ && doxygen doc/doxygen-java.conf', shell=True)
+subprocess.call('cd ../ && doxygen doc/doxygen-dotnet.conf', shell=True)
```

Comment:
I know this was already like this, but `subprocess.call`, as opposed to `subprocess.check_call`, silently swallows errors. We should probably use the latter here.